vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/logcat/LogcatCollector.kt

```diff
@@ -79,6 +94,11 @@ class LogcatCollector : LogcatEventsListener, LogReportProvider {
 
     private data class DeviceState(
         val currentBatchId: String? = null,
-        val currentTest: LogTest? = null
+        val currentTest: TestState? = null
+    )
+
+    private data class TestState(
+        val test: LogTest,
+        val processId: Int
```

Comment:
add process id for test state to filter crashes related only to the test execution